src/engineering-meeting/styles/engineering-meeting.css

```diff
@@ -22,24 +22,84 @@
    -moz-osx-font-smoothing: grayscale;
    justify-content: center;
    display: flex;
+   background-color: #f9f9f9;
+   animation: fadeIn 1s;
+}
+
+@keyframes fadeIn {
+   0% {
+      opacity: 0;
+   }
+   100% {
+      opacity: 1;
+   }
 }
 
-.engineeringMeeting .sketch-view {
+.engineeringMeeting .meeting-preview {
    display: flex;
    flex-direction: column;
+   align-self: center;
+   background-color: #f9f9f9;
+}
+
+.engineeringMeeting .meeting-preview .meeting-name {
+   position: relative;
+   padding-left: 1vmin;
+   text-align: center;
+   background-color: #f9f9f9;
+}
+
+.engineeringMeeting .meeting-preview .preview-images {
+   display: flex;
+   flex-direction: row;
+   vertical-align: middle;
+   width: fit-content;
+   height: fit-content;
+   box-shadow: 7px 7px 7px #00000077;
+   border-style: groove;
+   border-left-width: 1px;
+   border-top-width: 2px;
+   border-bottom-width: 2px;
+   border-right-width: 2px;
+   border-radius: 30px;
+   border-color: #030303;
+   overflow: hidden;
+   -webkit-mask-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAA5JREFUeNpiYGBgAAgwAAAEAAGbA+oJAAAAAElFTkSuQmCC); /* this fixes the overflow:hidden in Chrome/Opera */
```

Comment:
This two weird and long properties are to make sure that the content of the image doesn't get visible behind the are of the border for the images that we are defining (as the border has round corners)